bastion_current_ip.py

```diff
@@ -0,0 +1,27 @@
+# obtain current public ip, create a variables.tfvars file 
```

Comment:
I would write the meat of this script in a function, then call this under something like this: https://docs.python.org/3/library/__main__.html .  That way you can make your code more reusable, since you can call this script also as a library.